tests/proxy.rs

```diff
@@ -132,7 +132,7 @@ async fn test_tls(acceptor: TlsAcceptor, connector: TlsConnector) -> Result<(),
         Ok(()) as Result<(), IoError>
     };
 
-    let proxy = flv_tls_proxy::start(PROXY, acceptor.clone(), SERVER.to_string());
+    let proxy = flv_tls_proxy::start(PROXY, acceptor.clone(), SERVER.to_string(), false);
 
     let _ = zip(proxy, zip(client_ft, server_ft)).await;
 
```

Comment:
probably encapsulate all auth related into separate structure and pass that instead of inlining init proxy.  later we can use trait to make implementation independent

Reply:
done